tuned/hardware/inventory.py

```diff
@@ -54,6 +54,14 @@ def _handle_udev_event(self, event, device):
 		if not device.subsystem in self._subscriptions:
 			return
 
+		retry = consts.HOTPLUG_WAIT_FOR_DEV_INIT_RETRIES
+		while not device.is_initialized and retry > 0:
+			log.debug("Device '%s' is uninitialized, waiting '%d' seconds for its initialization." % (device, consts.HOTPLUG_WAIT_FOR_DEV_INIT_DELAY))
```

Comment:
Maybe print this as float, the current log would just say 0 seconds in the default configuration.
```suggestion
			log.debug("Device '%s' is uninitialized, waiting '%.2f' seconds for its initialization." % (device, consts.HOTPLUG_WAIT_FOR_DEV_INIT_DELAY))
```

Reply:
Thanks, fixed.